cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md

```diff
@@ -551,3 +670,5 @@ Paul Petefish
 Eric Sheridan - eric.sheridan@owasp.org
 
 Dominique Righetto - dominique.righetto@owasp.org
+
+[Arun John Kuruvilla (Amazon Application Security Team)](https://github.com/arunjohnkuruvilla)
```

Comment:
Please do not add your name because we want to use GH feature for contributors. This sections will be deleted near the end of this year - see issue #91.

Reply:
Removed.